docs/MainSite/Installation.md

```diff
@@ -167,21 +168,21 @@ or (equivalently) use the `module load` command.
 it is recommended you read the [documentation](https://spack.readthedocs.io) if
 you require features such as packages installed with different compilers.
 
-### Building SpECTRE
+### Building SpECTRE (within Spack environment)
 
 After the dependencies have been installed, Charm++ and SpECTRE can be compiled.
 Follow these steps:
 
 1.  Clone [SpECTRE](https://github.com/sxs-collaboration/spectre) into
-    `SPECTRE_ROOT`, a directory of your choice.
+    `$SPECTRE_ROOT`, a directory of your choice.
```

Comment:
Assuming people installing with Spack didn't previously install with Docker, this variable needs to be exported before it can be used.